tests/data_context/test_workspace_aware_context.py

```diff
@@ -0,0 +1,435 @@
+"""
```

Comment:
I did not test reading the the workspace id from a config file. I've only tested getting the parameter value from a env var or from passing it in directly. No logic changed that reads values from the config file.